interfaces/Portalicious/src/app/components/page-layout/components/registration-header/registration-header.component.html

```diff
@@ -0,0 +1,32 @@
+<p-card
+  styleClass="[&_.p-card-content]:flex-end group relative h-full [&_.p-card-title]:flex-grow"
+>
+  <div class="flex">
+    <h2>{{ registration.data()?.name }}</h2>
+    <p-button
+      label="Add note"
+      i18n-label="@@add-note"
+      rounded
+      outlined
+      icon="pi pi-pen-to-square"
+      (click)="openAddNoteDialog()"
+      class="ml-auto"
+    />
+  </div>
+  <app-data-list [data]="registrationData()" />
+  <div class="mt-4 text-end">
+    <span class="text-grey-700 txt-system-s">
+      @if (registration.isPending()) {
+        <app-skeleton-inline width="8rem" />
+      }
+      <ng-container i18n>Registered: </ng-container>
+      @if (registration.data()?.registrationCreatedDate) {
+        {{ registration.data()?.registrationCreatedDate | date: 'dd-MM-yyyy' }}
+      } @else {
+        <ng-container i18n="@@generic-not-available"
+          >Not available</ng-container
```

Comment:
In other places, the "Not available label" has a `text-grey-500` class on it. And also, we're defining/reusing this copy label in several places now. It makes me wonder whether we should have a `NotAvailableLabelComponent` which simply renders

```html
<span
    i18n="@@generic-not-available"
    class="text-grey-500"
    >Not available</span
```